components/XmtpProvider.tsx

```diff
@@ -68,7 +75,10 @@ export const XmtpProvider: React.FC = ({ children }) => {
     const streamConversations = async () => {
       const stream = await client.conversations.stream()
       for await (const convo of stream) {
-        if (convo.peerAddress !== walletAddress) {
+        if (
```

Comment:
@neekolas The v7.0.0 release notes did not advise this. Is it necessary to do?

Reply:
Yes. We should add that to the notes